core/src/test/java/io/cucumber/core/options/CucumberPropertiesParserTest.java

```diff
@@ -44,14 +43,9 @@ public boolean addClass(Class<?> glueClass) {
         }
 
         @Override
-        public void start() {
-
-        }
+        public void start() {}
```

Comment:
Please keep these separated. The next auto format with IDEA will just revert it.

Reply:
Done